src/Symfony/Component/Form/Tests/Extension/Core/DataTransformer/UuidToStringTransformerTest.php

```diff
@@ -56,15 +56,6 @@ public function testReverseTransformEmpty()
         $this->assertNull($reverseTransformer->reverseTransform(''));
     }
 
-    public function testReverseTransformExpectsString()
-    {
-        $reverseTransformer = new UuidToStringTransformer();
-
-        $this->expectException(TransformationFailedException::class);
-
-        $reverseTransformer->reverseTransform(Uuid::fromString('123e4567-e89b-12d3-a456-426655440000')->toBase32());
-    }
```

Comment:
I think this is a fortunate side-effect, where this data transformer also allows the new representations (thus removing this test case)